android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdView.java

```diff
@@ -108,19 +116,84 @@ public void setCustomData(@Nullable final String value)
         customData = value;
     }
 
-    public void setExtraParameters(@Nullable final ReadableMap readableMap)
+    public void setExtraParameters(@Nullable final ReadableArray readableArray)
     {
-        if ( readableMap != null )
+        if ( readableArray == null ) return;
+
+        Map<String, Object> extraParametersMap = new HashMap<>();
+
+        for ( int i = 0; i < readableArray.size(); i++ )
```

Comment:
This logic is used in the other PR for native adviews - so is this something we can consolidate the logic for? Like in Utils class?

Reply:
Created a helper function to support both `AdView` and `NativeAdView`.